x-pack/plugin/core/template-resources/src/main/resources/logs-otel@mappings.json

```diff
@@ -0,0 +1,97 @@
+{
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "dynamic": "strict",
+      "properties": {
+        "data_stream.type": {
+          "type": "constant_keyword",
+          "value": "logs"
+        },
+        "data_stream.dataset": {
+          "type": "constant_keyword"
+        },
+        "data_stream.namespace": {
+          "type": "constant_keyword"
+        },
+        "@timestamp": {
+          "type": "date_nanos"
+        },
+        "observed_timestamp": {
+          "type": "date_nanos"
+        },
+        "severity_number": {
+          "type": "byte"
+        },
+        "severity_text": {
+          "type": "keyword"
+        },
```

Comment:
I believe our mapping spec says these will be `log.level` and `log.level_number`. That said I'm not opposed to going full on towards the OTLP names

Reply:
I think we should align with OTLP all the way here. We can have an alias field for `log.level`.

Reply:
The spec is changed now